content/blog/architect-aws-application-infra-with-pulumi-stack-references/index.md

```diff
@@ -452,5 +452,5 @@ information between these projects. You can find a runable version of this infra
 
 This demonstrates the suggested way of architecting Pulumi applications both for operability and collaboration.
 
-You can try this code today get started with Pulumi today at [https://www.pulumi.com](https://www.pulumi.com) and
-following the [Getting Started](https://www.pulumi.com/docs/get-started/aws/) for managing AWS resources with Pulumi.
+You can try this code today get started with Pulumi today at [https://www.pulumi.com](/) and
```

Comment:
Does this need a `relref`?

Reply:
There's really no need for `relref` here. The root isn't going to change.